preload/single-download-late-used-preload.html

```diff
@@ -2,16 +2,18 @@
 <title>Ensure preloaded resources are not downloaded again when used</title>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src="/preload/resources/preload_helper.js"></script>
 <link rel=preload href="resources/square.png?pipe=trickle(d1)" as=image>
 <script>
     var link = document.getElementsByTagName("link")[0]
     assert_equals(link.as, "image");
     link.addEventListener("load", () => {
-        assert_equals(performance.getEntriesByType("resource").length, 3)
+        verifyNumberOfDownloads("resources/square.png?pipe=trickle(d1)", 1);
         var img = document.createElement("img");
         img.src = "resources/square.png?pipe=trickle(d1)";
         img.onload = () => {
-            assert_equals(performance.getEntriesByType("resource").length, 3);
+            verifyPreloadAndRTSupport();
```

Comment:
I prefer to explicitly verify feature support in each test, as not all of them use `verifyNumberOfDownloads`